### PR TITLE
feat: Drop the use of the Tubular repository

### PR DESCRIPTION
**Pull request by mrtmm**
The Tubular repository has been deplrecated and the relevant scripts have been moved to the edx-platform codebase.

### CODE REVIEW

**Comment by fghaas** [comment]
@mrtmm, If you add `Fixes: #32` at the end of your commit message, then that issue will auto-close once we merge this commit to `main`.

**Comment by mrtmm** [comment]
Ah, nice, thanks!

**Comment by mrtmm** [comment]
Hmm, maybe it will be good to cut one more 3.x release in case someone needs the latest fixes in `quince`?

**Comment by fghaas** [comment]
> Hmm, maybe it will be good to cut one more 3.x release in case someone needs the latest fixes in `quince`?

Yes, sounds good. Please go ahead and do that.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ appropriate one:
 | Olive            | `>=15.0, <16`     | `main`        | 2.x.x          |
```
Comment (mrtmm) [inline README.md:21]: @fghaas should we create release named branches here now until `Quince`?

File: tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cool_off_days=$1
+pip install --upgrade pip && pip install -r scripts/user_retirement/requirements/base.txt --exists-action w
```
Comment (fghaas) [inline tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh:2]: Just so I understand this correctly: retirement has moved into the platform repo, but the packages needed for retirement don't actually get baked into the default Tutor `openedx` image, meaning we have to install them every time we run the retirement scripts?

**Review by fghaas — COMMENTED** [review]
Looks good to me in general; I just have one question.

File: tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh (hunk 2 above)
Comment (mrtmm) [reply tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh:2]: Yes, those requirements are under the `/scripts/user_retirement` directory and not installed in the `openedx` image.

Comment (fghaas) [reply tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh:2]: OK, doesn't that run counter to the idea of having pre-built container images though?

(In other words: should we perhaps continue to build images, just as we did prior to Redwood?)

Comment (mrtmm) [reply tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh:2]: Hmm, okay, I guess we can do that also yes.

Comment (mrtmm) [reply tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh:2]: Should I go ahead and update this with building the image?

Comment (fghaas) [reply tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh:2]: I think so, yes.

**Review by fghaas — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**